src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 -----
 
 * `AbstractNormalizer::handleCircularReference` is now final, and receives two optional extra arguments: the format and the context
+* `YamlEncoder` now handle the `yml` format too
```

Comment:
`extension` here is more appropriate. YAML is a format and `.yaml`/`.yml`are two possible file extensions.

Reply:
but here we intro an actual new format :)

Reply:
?!

Reply:
It's about what string represents the format. Nothing to do with the extension. Check out the issue https://github.com/symfony/symfony/issues/28768